<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>现在是</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您已熬了</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>36</pointsize>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>分钟</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>641</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string>熬夜会对身体造成多种损害：
1.	经常疲劳，免疫力下降：人经常熬夜造成的后遗症，最严重的就是疲劳、
2.	精神不振；人体的免疫力也会跟着下降，感冒、胃肠感染、过敏
等等自律神经失调症状都会出现。
3.	头痛：熬夜的隔天，上班或上课时经常会头痛脑胀、注意力无法集中
，甚至会出现头痛的现象，
长期熬夜、失眠对记忆力也有无形的损伤。
4.	黑眼圈、眼袋：夜晚是人体的生理休息时间，该休息而没有休息，就会因为过劳，造成眼睛周围的血液循环不良，
引起黑眼圈、眼袋或是白眼球布满血丝。
6.皮肤干燥、长黑斑和青春痘：晚上11时到第二天凌晨3时是美容时间，也是人体的经脉运行到
胆、肝的时段。这两个器官如果没有获得充分的休息，就会表现在皮肤上，
容易出现粗糙、脸色偏黄、长黑斑和青春痘等问题。
对于习惯早睡的人来说，最迟也要在凌晨1时的养肝时间进入熟睡期。
而且，更糟糕的是，长期熬夜会慢慢地出现失眠、健忘、易怒、焦虑不安等神经、精神症状。
7.肝功能异常的比例较一般人高。
8.罹患慢性病的概率增加：熬夜族的肾上腺素等激素分泌量也比一般人高，使新陈代谢的压力增加，
进而产生慢性疾病。如高血压、糖尿病等。
9.视力危害最大，建议每熬夜（23：00以后）1小时，做一次眼保健操。
否则后果严重。一定要注意，戴眼镜是很痛苦的.如果长期熬夜，
更会慢慢地出现失眠、健忘、易怒、焦虑不安等神经、精神症状。过度劳累使身体的神经系统功能紊乱，引起体内主要的器官
和系统失衡，
比如发生心律不齐、内分泌失调等等，严重的就会导致全身的应激状态、感染疾病的概率相应提高。
疲劳症状强烈的人比一般人患上呼吸、消化系统、循环器官等各种感染症的机会也增加许多。

</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>您知道吗?  Did you know?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>您希望几分钟后提醒您?</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>请合理设置,为自己的健康负责</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
